src/dbwrapper.h

```diff
@@ -44,6 +45,11 @@ const std::vector<unsigned char>& GetObfuscateKey(const CDBWrapper &w);
 
 };
 
+class CBitcoinLevelDBLogger : public leveldb::Logger {
```

Comment:
well, h for declaration and cpp for implementation, at least that is what I always thought. You mean just putting the class directly in the cpp ?